--- ./pulseaudio-17.0/src/pulse/sample.h	2024-01-12 19:22:09.000000000 +0200
+++ ./pulse/sample.h	2026-06-04 21:12:58.883909552 +0300
@@ -280,13 +280,13 @@
 /** Calculate the time it would take to play a buffer of the specified
  * size with the specified sample type. The return value will always
  * be rounded down for non-integral return values. */
-pa_usec_t pa_bytes_to_usec(uint64_t length, const pa_sample_spec *spec) PA_GCC_PURE;
+typedef pa_usec_t (__cdecl *pa_bytes_to_usec)(uint64_t length, const pa_sample_spec *spec) PA_GCC_PURE;
 
 /** Calculates the size of a buffer required, for playback duration
  * of the time specified, with the specified sample type. The
  * return value will always be rounded down for non-integral
  * return values. \since 0.9 */
-size_t pa_usec_to_bytes(pa_usec_t t, const pa_sample_spec *spec) PA_GCC_PURE;
+typedef size_t (__cdecl *pa_usec_to_bytes)(pa_usec_t t, const pa_sample_spec *spec) PA_GCC_PURE;
 
 /** Initialize the specified sample spec and return a pointer to
  * it. The sample spec will have a defined state but
