--- ./pulse/sample.h	2024-01-12 19:22:09.000000000 +0200
+++ ./work/sample.h	2026-06-05 17:06:58.327920922 +0300
@@ -23,7 +23,9 @@
 
 #include <inttypes.h>
 #include <sys/types.h>
+#ifndef _MSC_VER
 #include <sys/param.h>
+#endif
 
 #include <pulse/gccmacro.h>
 #include <pulse/cdecl.h>
@@ -280,13 +282,13 @@
 /** Calculate the time it would take to play a buffer of the specified
  * size with the specified sample type. The return value will always
  * be rounded down for non-integral return values. */
-pa_usec_t pa_bytes_to_usec(uint64_t length, const pa_sample_spec *spec) PA_GCC_PURE;
+typedef pa_usec_t (__cdecl *pa_bytes_to_usec)(uint64_t length, const pa_sample_spec *spec) PA_GCC_PURE;
 
 /** Calculates the size of a buffer required, for playback duration
  * of the time specified, with the specified sample type. The
  * return value will always be rounded down for non-integral
  * return values. \since 0.9 */
-size_t pa_usec_to_bytes(pa_usec_t t, const pa_sample_spec *spec) PA_GCC_PURE;
+typedef size_t (__cdecl *pa_usec_to_bytes)(pa_usec_t t, const pa_sample_spec *spec) PA_GCC_PURE;
 
 /** Initialize the specified sample spec and return a pointer to
  * it. The sample spec will have a defined state but
