--- ./build/pulse/operation.h	2024-01-12 19:22:09.000000000 +0200
+++ ./work/operation.h	2026-06-04 20:58:06.391919768 +0300
@@ -36,20 +36,20 @@
 typedef void (*pa_operation_notify_cb_t) (pa_operation *o, void *userdata);
 
 /** Increase the reference count by one */
-pa_operation *pa_operation_ref(pa_operation *o);
+typedef pa_operation* (__cdecl *pa_operation_ref)(pa_operation *o);
 
 /** Decrease the reference count by one */
-void pa_operation_unref(pa_operation *o);
+typedef void (__cdecl *pa_operation_unref)(pa_operation *o);
 
 /** Cancel the operation. Beware! This will not necessarily cancel the
  * execution of the operation on the server side. However it will make
  * sure that the callback associated with this operation will not be
  * called anymore, effectively disabling the operation from the client
  * side's view. */
-void pa_operation_cancel(pa_operation *o);
+typedef void (__cdecl *pa_operation_cancel)(pa_operation *o);
 
 /** Return the current status of the operation */
-pa_operation_state_t pa_operation_get_state(const pa_operation *o);
+typedef pa_operation_state_t (__cdecl *pa_operation_get_state)(const pa_operation *o);
 
 /** Set the callback function that is called when the operation state
  * changes. Usually this is not necessary, since the functions that
